packages/ui-extensions/src/surfaces/point-of-sale/api/navigation-api/navigation-api.ts

```diff
@@ -1,12 +1,14 @@
 export interface NavigationApiContent {
-  /** Navigate to a route in current navigation tree.
+  /**
+   * Using `(screenName: string, params?: any) => void` navigates to a route in current navigation tree.
```

Comment:
Both should be Promise< void > since we are overloading.

```suggestion
   * Using `(screenName: string, params?: any)` navigates to a route in current navigation tree.
```